jumpscale/clients/explorer/models.py

```diff
@@ -563,6 +574,15 @@ def resource_units(self):
             resource_units.sru += self.size
         return resource_units
 
+class PublicIP(Base):
+    id = fields.Integer()
+    ipaddress = fields.IPRange()
```

Comment:
why ip range and not an ip address?

Reply:
Because the ip is defined as a range. We need to know the range in zos.

For example: `10.10.10.10/16` can't fit in type Ipaddress